west.yml

```diff
@@ -46,7 +46,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: v2.3.0-rc1-ncs1-rc1
+      revision: pull/314/head
```

Comment:
This nrf PR and the zephyr PR need a rebase onto the new zephyr history.

Reply:
both PR rebased.